src/components/UI/atoms/Link/Link.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { fontSizes } from '<variables>/fonts';
+
+/**
+ * @description - Link Component
+ *
+ * @prop {boolean} isTag
+ * @prop {string} fontSize
+ * @prop {string} href
+ * @prop {children} children - children
+ *
+ * @return {component} Link
+ */
+const Link = ({
+  href,
+  isTag,
+  fontSize,
+  margin,
+  onClick,
+  children,
+}) => (
+  <Link.Container
+    href={href || '#'}
+    isTag={isTag}
+    fontSize={fontSize}
+    margin={margin}
+    onClick={onClick}
+  >
+    {children}
+  </Link.Container>
+);
+
+Link.propTypes = {
+  children: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+    PropTypes.element,
+  ]).isRequired,
+  href: PropTypes.string,
+  isTag: PropTypes.bool,
+  fontSize: PropTypes.oneOf(Object.keys(fontSizes)),
+  onClick: PropTypes.func,
+  margin: PropTypes.string,
+};
+
+Link.defaultProps = {
+  display: 'inline-block',
+  fontSize: 'normal',
+  margin: '0',
+};
+
+Link.Container = styled.a`
+  ${({
+    theme,
+    isTag,
+    fontSize,
+    margin,
+  }) => `
+    padding: 0;
```

Comment:
I expect logic for internal and external links. e.g. links to other sites vs links to places in your app. Something link this
```
 return isExternal ? (
    <TextLink.External {...linkProps} href={to} target={'_blank'}>
      {innerContainer}
    </TextLink.External>
  ) : (
    <TextLink.Internal {...linkProps} to={to}>
      {innerContainer}
    </TextLink.Internal>
  )
```